lib/runtime/extrinsic/unchecked_extrinsic.go

```diff
@@ -0,0 +1,236 @@
+// Copyright 2020 ChainSafe Systems (ON) Corp.
+// This file is part of gossamer.
+//
+// The gossamer library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The gossamer library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the gossamer library. If not, see <http://www.gnu.org/licenses/>.
+package extrinsic
+
+import (
+	"math/big"
+
+	"github.com/ChainSafe/gossamer/lib/common"
+	"github.com/ChainSafe/gossamer/lib/crypto"
+	"github.com/ChainSafe/gossamer/lib/scale"
+)
+
+// TODO determine how to get these values from the Runtime
+const specVersion uint32 = 193 // encoded as additional singed data when building UncheckedExtrinsic
+//const transactionVersion uint32 = 1 // encoded as additional singed data when building UncheckedExtrinsic
```

Comment:
I think you can get these from the `core_version` runtime call

Reply:
Good idea, I didn't feel right hard coding this.